Mathlib/CategoryTheory/MorphismProperty.lean

```diff
@@ -695,6 +695,46 @@ theorem bijective_respectsIso : (MorphismProperty.bijective C).RespectsIso :=
 
 end Bijective
 
+/-- Typeclass expressing that a morphism property contain identities. -/
+class ContainsIdentities (W : MorphismProperty C) : Prop :=
+  /-- for all `X : C`, the identity of `X` satisfies the morphism property -/
+  id_mem' : ∀ (X : C), W (𝟙 X)
+
+lemma id_mem (W : MorphismProperty C) [W.ContainsIdentities] (X : C) :
+  W (𝟙 X) := ContainsIdentities.id_mem' X
+
+namespace ContainsIdentities
+
+instance op (W : MorphismProperty C) [W.ContainsIdentities] :
+    W.op.ContainsIdentities := ⟨fun X => W.id_mem X.unop⟩
+
+lemma unop (W : MorphismProperty Cᵒᵖ) [W.ContainsIdentities] :
```

Comment:
I think this could be an instance, and there should be two more lemmas `of_op` and `of_unop`, like
```lean
lemma of_op (W : MorphismProperty C) [W.op.ContainsIdentities] :
    W.ContainsIdentities := ⟨fun X => W.op.id_mem (Opposite.op X)⟩ 
```
The same applies to `IsMultiplicative` below.